chapters/features.txt

```diff
@@ -5868,7 +5870,7 @@ s| Format
 <<<
 
 [[features-formats-mandatory-features-depth-stencil]]
-.Mandatory format support: depth/stencil with elink:VkImageType ename:VK_IMAGE_TYPE_2D
+.Mandatory format support: depth/stencil with ename:VkImageType ename:VK_IMAGE_TYPE_2D
```

Comment:
Because a link inside a link generates invalid html.

Reply:
Seems like an asciidoctor limitation, works fine in the PDF output. In any event, please use etext: for this instead of ename:, as these are enumerated type names, not enumerant names.